fdbclient/LinkTest.cpp

```diff
@@ -0,0 +1 @@
+int main() { return 0; }
```

Comment:
Please add code documentation explaining the purpose of the file? (Empty main() to ensure that no cross module dependencies exist for the module etc.)